wwwroot/test/csv/regionMapping.json

```diff
@@ -0,0 +1,197 @@
+{
+    "comments": "Matching takes place in the order defined in this file. Place code matches before name matches, and smaller regions before larger ones.",
```

Comment:
>As you probably know, JSON keys are unordered

Yeah, good point. The truth seems to be a lot murkier than that simple statement. To fix it will require backwardly incompatible changes so I won't try to squeeze it in here. https://github.com/TerriaJS/terriajs/issues/930